english/challenge-marshmellow-to-rock-off/tenacious-d.md

```diff
@@ -0,0 +1,30 @@
+Realizing now that the marshmallows have become self-aware, much unlike SkyNet, 
+your jacket reacts to these sentient beings and springs to life almost destroying 
+your skin in the process. The jacket now forms a leather mouth, through which you 
+hear the sound of none other than Ronnie James Dio, the guitar legend.
+
+" I heard you brave young Jacket-guy, you are hungry for the rock 
+But to learn the ancient methods, secret doors you must unlock 
+Escape your father's clutches in this oppressive neighborhood 
+On a journey you must go to find the land of Hollywood 
+In the City of Fallen Angels, where the ocean meets the sand 
+You will form a strong alliance and the world's most awesome band 
+To find your fame and fortune, through the valley you must walk 
+You will face your inner demons, now go my son and ROCK!"
+
+As the epic monologue ends, you realize that the lighter and fork and rope 
+you hold must be in some way related to the cryptic message that Dio said.
+As you ponder this, you realize the marshmallows are still awestruck by Dio's 
+awesomeness. You take this chance to craft a makeshift Fire whip by burning the 
+rope and tying it to the fork and whiplashing around you like Indiana Jones and 
+harge through the weak wall of Marshmallows.
+
+As you scurry forward screaming your battle cry in sync to Dio's vocals, you see 
+a small wooden door directly ahead of you saying "Keep Out" in faded lettering. 
+To your immediate right, a dim light shines not far from your current position, 
+as you gleam towards it while darting towards the door, you arrive at a tiny 
+(proverbial) fork in the road, will you:
+
+(Go towards the light?)[light-route/joker.md]
```

Comment:
Did you mean for this to be a link?  If so, your syntax is backwards.  The text should be in square brackets and the link in parentheses rather than vice versa.  You can view the page on GitHub on your fork to check if the link is displaying correctly.